src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Parser/ConfigurationBinder.cs

```diff
@@ -218,7 +231,7 @@ private void ParseGetValueInvocation(BinderInvocation invocation)
                         return;
                     }
 
-                    ITypeOfOperation? typeOfOperation = operation.Arguments[1].ChildOperations.FirstOrDefault() as ITypeOfOperation;
+                    ITypeOfOperation? typeOfOperation = GetArgumentForParameterAtIndex(operation.Arguments, 1).ChildOperations.FirstOrDefault() as ITypeOfOperation;
```

Comment:
The same issue found in `GetValue(...)` invocations, but could not test. Not sure if it is known issue or it is by design, but the generator is not generating the source for any test case I tried. CC @layomia

Reply:
Apparently `GetValue` only works for primitive types

Reply:
Added corresponding test